funnel/templates/comments.html.jinja2

```diff
@@ -12,22 +12,24 @@
           </button>
           <button class="js-uncollapse mui-btn mui-btn--nostyle mui--hide" aria-expanded="false" aria-label="{% trans %}Collapse comment{% endtrans %}">
             {{ faicon(icon='caret-circle-down', icon_size='subhead') }}
-          </button>&nbsp;
-          {% if comment.state.DELETED -%}
-            <span class="commenter">[{% trans %}deleted{% endtrans %}]</span>
-          {%- else -%}
-            <span class="commenter"><strong>{{ comment.user.pickername }}</strong></span>
-            {%- if comment.user == document.user %}
-              <span class="badge">Proposer</span>
-            {% endif %}
-            {%- if 'crew' in project.roles_for(comment.user) %}
-              <span class="badge">Crew</span>
-            {% endif %}
-            <span class="mui--text-caption mui--text-accent">{{ comment.created_at|age }}</span>
-            {%- if comment.edited_at %}
-              <span class="mui--text-caption mui--text-accent">({% trans %}edited{% endtrans %} {{ comment.edited_at|age }})</span>
+          </button>
+          <div class="comment--header__details">
+            {% if comment.state.DELETED -%}
+              <span class="commenter">[{% trans %}deleted{% endtrans %}]</span>
+            {%- else -%}
+              <span class="commenter"><strong>{{ comment.user.pickername }}</strong></span>
+              {%- if comment.user == document.user %}
```

Comment:
Unrelated to this PR - `document.user` is only available if `document` is a Proposal. We're also using `commenttree()` in project comments page where `document` is a Commentset.

Reply:
Comment have a `badges` set that it populates based on specific roles being available on the parent.